src/lib/y2network/connection_config/bonding.rb

```diff
@@ -34,6 +35,31 @@ def initialize
         @slaves = []
         @options = ""
       end
+
+      # @see Y2Network::InterfacesCollection
+      #
+      # @param interfaces [Y2Network::InterfacesCollection]
+      def update_interfaces!(interfaces)
+        return if slaves.empty?
+        new_slaves = []
+
+        slaves.each do |index, slave|
+          alternative = interfaces.by_name(slave)
+          unless alternative
+            alternative = Y2Network::FakeInterface.new(slave)
```

Comment:
So we are modifying the interfaces collection here? I would expect all the interfaces to be created at `InterfacesReader`, but not here. We should check and document this behavior

Reply:
Yep, probably moving it to the interfaces reader will be better, so having to know or being responsible of replacing the name of interfaces related by Y2Network::Interface objects is something it could deal with